### PR TITLE
[#35] Log all user actions (once all functionality is ready)

### PR DESCRIPTION
**Pull request by Sheptytskyid**

### CODE REVIEW

File: src/main/java/com/manardenza/controller/HotelController.java [hunk 1]
```diff
@@ -11,25 +13,35 @@
...
         return hotelService.findHotelByName(hotelName);
     }
 
     public List<Hotel> findHotelByCity(String cityName) {
+        log.info(String.format("User %s has searched for hotels by city %s", currentUser.getUser().getFullName(),
+            cityName));
         return hotelService.findHotelByCity(cityName);
     }
 
+    //TODO: consider deleting this method if it won't be used in console
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/controller/HotelController.java:36]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)

File: src/main/java/com/manardenza/controller/ReservationController.java [hunk 2]
```diff
@@ -3,28 +3,50 @@
...
+            log.info(String.format("%s has successfully cancelled reservation %d", currentUser
+                .getUser().getFullName(), idReservation));
+        } else {
+            log.info(String.format("Reservation %d cancellation failure", currentUser
+                .getUser().getFullName(), idReservation));
+        }
+        return wasCancelled;
     }
 
+    //TODO: consider deleting this method if it won't be used in console
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/controller/ReservationController.java:49]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'

File: src/main/java/com/manardenza/entity/User.java [hunk 3]
```diff
@@ -12,4 +12,10 @@ public User(String firstName, String lastName) {
         this.firstName = firstName;
         this.lastName = lastName;
     }
+
+    public String getFullName() {
+        StringBuilder fullName = new StringBuilder(firstName);
```
Comment (zozich) [inline src/main/java/com/manardenza/entity/User.java:17]: It is okay to use "+" concatenation. It anyway creates StringBuilder underneath

**Review by zozich — APPROVED** [review]
Ok. See my comment

**Pull request merged**